app/models/concerns/aggregate_associations.rb

```diff
@@ -0,0 +1,16 @@
+# typed: false
+# frozen_string_literal: true
+
+# Provides a helper method for associations
```

Comment:
Would you consider expanding the scope of this comment to say more about how/why we need it?